File: src/main/resources/static/js_file/model/address.js
import {getElement} from "../helperMethod.js";

 class Address {
    constructor(type) {
        this.type = type;
        this.fullname = getElement(`#${this.type}_fullname`).value;
        this.zipcode = getElement(`#${this.type}_zipcode`).value;

        this.country = getElement(`#${this.type}_country`).value;
        this.apt = getElement(`#${this.type}_apt`).value;
        this.street = getElement(`#${this.type}_street`).value;
        this.city = getElement(`#${this.type}_city`).value;
        this.state = getElement(`#${this.type}_state`).value;
    }

}
export {Address}

File: src/main/java/com/art/jeanyvesart/model/Address.java
package com.art.jeanyvesart.model;

import lombok.Data;

import java.io.Serializable;

@Data
public class Address implements Serializable {
    private static final long serialVersionUID = 1L;

    private Long id;
    private String country;
    private String fullname;
    //@NotBlank(message = "Delivery street is required")

    private String street;
    //@NotBlank(message = "Delivery city is required")

    private String city;
    //@NotBlank(message = "Delivery state is required")
    private String state;
    //@NotBlank(message = "Zip code is required")
    private String apartment;
    private String zip;
}